--- a/include/wx/gtk/private/wayland.h
+++ b/include/wx/gtk/private/wayland.h
@@ -19,7 +19,7 @@
 // Use Wayland protocols directly if we have wayland-client and a new enough
 // GTK version to compile the code using it.
 #if defined(GDK_WINDOWING_WAYLAND) && GTK_CHECK_VERSION(3,20,0)
-    #ifdef wxHAVE_WAYLAND_CLIENT
+    #if wxHAVE_WAYLAND_CLIENT
         #define wxHAVE_WAYLAND_PROTOCOLS
     #endif
 #endif
